app/test_main.py

```diff
@@ -1,3 +1,61 @@
+import pytest
+
 from app.main import get_human_age
 
-# write your code here
+
+@pytest.mark.parametrize(
+    "cat_age,dog_age,human_ages",
+    [
+        pytest.param(
+            14, 15, [0, 1],
+            id="first 15 cat or dog years should be equal to 1 human year"
+        ),
+        pytest.param(
+            24, 23, [2, 1],
+            id="next 9 cat or dog years should give 1 more human year"
+        ),
+        pytest.param(
+            28, 28, [3, 2],
+            id="every 4 next cat years should give 1 extra human year"
+        ),
+        pytest.param(
+            28, 29, [3, 3],
+            id="every 5 next dog years should give 1 extra human year"
+        ),
+        pytest.param(
+            0, 0, [0, 0],
+            id="should return zeros when cat or dog age equals zeros"
+        ),
+    ]
```

Comment:
add more tests:
In ideal scenario, you should test for all the boundaries between different outputs, for example, for cat years:

0;
14: largest possible value that returns 0 human years;
15: smallest possible value that returns 1 human year;
23: largest possible value that returns 1 human year; and so on.
This does not apply to numbers past 28, since they increase linearly, and you can just test for a large value.